src/main/java/thePackmaster/actions/wardenpack/PierceVeilAction.java

```diff
@@ -0,0 +1,107 @@
+package thePackmaster.actions.wardenpack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.AbstractCard.CardType;
+import com.megacrit.cardcrawl.cards.CardGroup;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.screens.CardRewardScreen;
+import com.megacrit.cardcrawl.vfx.cardManip.ShowCardAndAddToDiscardEffect;
+import com.megacrit.cardcrawl.vfx.cardManip.ShowCardAndAddToHandEffect;
+import thePackmaster.util.Wiz;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+
+public class PierceVeilAction extends AbstractGameAction {
```

Comment:
Did you copy this from modargo? If so, his pack is merged so you could use that instead of copying the same class again